meeshkan/serve/mock/faker/schema_faker.py

```diff
@@ -0,0 +1,275 @@
+########################
+#### FAKER
```

Comment:
Maybe a docstring at the top of the module could be more useful than this?

Reply:
That was carefully preserved from the first faker implementation as a part of history. :)